### ISSUE TITLE
clldutils.dsv

### ISSUE THREAD

**Issue by laiyunfan** [lingpy/poepy#2]
Poepy requires the module clldutils. However clldutils.dsv is removed from the module recently, as far as I know.
When I do `from poepy import *`, it gives me:

> ---> 15 from clldutils.dsv import reader
ModuleNotFoundError: No module named 'clldutils.dsv'

Any solution?

**Comment by xrotwang** [comment]
Yep, solution is to also require `csvw` and import `from csvw.dsv import reader`.

**Comment by xrotwang** [comment]
It looks like there are a couple more issues. E.g. `poepy` not listing all dependencies in `setup.py`. I'll have a look and hope to push out a new release within the week.

**Comment by LinguList** [comment]
Thanks, @xrotwang!

**Comment by xrotwang** [comment]
@LinguList looks like this requires a new release of `sinopy`, too:
```
ImportError while importing test module '/home/forkel/venvs/lingpy/poepy/tests/test_poem.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_poem.py:2: in <module>
    from poepy.poem import *
src/poepy/__init__.py:1: in <module>
    from .poepy import *
src/poepy/poepy.py:9: in <module>
    from sinopy.util import is_chinese
E   ImportError: No module named 'sinopy.util'
```

**Comment by LinguList** [comment]
It's just one function, called, is_chinese:

```python
"""
Utility functions that can be used without loading extensive amounts of data.
"""

def is_chinese(name):
    """
    Check if a symbol is a Chinese character.

    Note
    ----

    Taken from http://stackoverflow.com/questions/16441633/python-2-7-test-if-characters-in-a-string-are-all-chinese-characters
    """
    if not name:
        return False
    for ch in name:
        ordch = ord(ch)
        if not (0x3400 <= ordch <= 0x9fff) and not (0x20000 <= ordch <= 0x2ceaf) \
                and not (0xf900 <= ordch <= ordch) and not (0x2f800 <= ordch <= 0x2fa1f): 
                return False
    return True
```

I'd rather say: drop sinopy dependency, as sinopy needs to be re-done, and this function could even be placed into linse, right?

**Comment by xrotwang** [comment]
Yes, dropping sinopy as dependency would help a lot, since this seems to be yet another rabbit-hole which I don't necessarily want to go down right now :)

**Comment by LinguList** [comment]
hehe, me neither ;-)

**Comment by LinguList** [comment]
but the function is so small, it can even be placed into poepy, right?

**Comment by xrotwang** [comment]
Yes, that looks like the best option.

**Comment by xrotwang** [comment]
@laiyunfan just released `poepy` 0.2. Could you re-install via `pip install poepy==0.2` and try again?

**Comment by laiyunfan** [comment]
It works perfectly so far. Thanks!
